src/handlers/comment/handlers/wallet.ts

```diff
@@ -1,10 +1,12 @@
 import { upsertWalletAddress } from "../../../adapters/supabase";
 import { getBotContext } from "../../../bindings";
+import { addCommentToIssue } from "../../../helpers";
 import { Payload } from "../../../types";
 
 export const registerWallet = async (body: string): Promise<void> => {
   const { log, payload: _payload } = getBotContext();
   const payload = _payload as Payload;
+  const issue = payload.issue;
   const sender = payload.sender.login;
   const regex = /(0x[a-fA-F0-9]{40})/g;
```

Comment:
Would you also be able to handle ENS?

Reply:
It might make the most sense to ENS resolve at the moment of parsing. And if it resolves, just to save the raw address?

Reply:
This regex doesn't handle ENS

Reply:
I understand that, I'm saying to add it

Reply:
I wrote this the other day in normal javascript. It relies on some indie developer's Cloudflare function so please be sure to do error handling/logging properly in case they take down their endpoint.

https://github.com/ubiquity/pay.ubq.fi/blob/773947bad8fd069adf99fb223373fcbdfab4600d/static/scripts/render-transaction.js#L88

Reply:
In lastest commits, it should be handled